src/metrics/enhanced-metrics.ts

```diff
@@ -16,6 +17,10 @@ enum RuntimeTagValues {
   Node12 = "nodejs12.x",
 }
 
+export function getVersionTag(): string {
+  return `datadog_lambda:v${datadogLambdaVersion}`;
+}
```

Comment:
Small nit - maybe we should put this function in utils somewhere? Exporting a DD lambda version tag function from `enhanced-metrics.ts` seems a bit off.

Reply:
Great call out, I moved that to the parent folder 👍

Reply:
Oh I thought you meant importing the `datadogLambdaVersion` from trace constants. It was something I was thinking about. I think the export here is mainly for testing. But, this function is really not used anywhere else. `datadogLambdaVersion` is used in trace tags which are set as an object so it's not exactly the same